opensrp-app/src/main/java/org/smartregister/sync/intent/SyncIntentService.java

```diff
@@ -78,6 +86,45 @@ private void doSync() {
         }
     }
 
+    private void verifyAuthorization() {
+        org.smartregister.Context opensrpContent = CoreLibrary.getInstance().context();
+        String baseUrl = opensrpContent.configuration().dristhiBaseURL();
+        if (baseUrl.endsWith("/")) {
+            baseUrl = baseUrl.substring(0, baseUrl.lastIndexOf("/"));
```

Comment:
Why not use  `baseUrl.substring(0, baseUrl.length - 1);`?

Currently, it searches the string again for the last occurence of `/`, right?